### PR TITLE
feat: basic helm chart

### PR DESCRIPTION
**Pull request by 0xlildoudou**
Hi,

I have write a simple Helm Chart for Kubernetes integration because i couldn't find the deployment procedure in the documentation.
https://docs.freelabz.com/in-depth/deployment#kubernetes

It works for closed clusters (no port opening in the cluster). You therefore need to start a shell in the “secator-manager” pod.

I'm available on Twitter/X (@lildoudou_) for improvements :)

### CODE REVIEW

File: helm/templates/secator-manager.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: {{ .Values.secatorManager.name }}
+  namespace: {{ .Values.namespace }}
+  labels:
+    name: secator-manager
+spec:
+  containers:
+  - name: secator-namager
```
Comment (ocervell) [inline helm/templates/secator-manager.yaml]: namager -> manager ?

**Comment by ocervell** [comment]
Thanks for this ! Indeed we have some k8s manifests that we are using internally, but haven't had the time to clean them up for publishing yet.

Give me some time and I will review this ;)

File: helm/templates/secator-manager.yaml (hunk 1 above)
Comment (0xlildoudou) [reply helm/templates/secator-manager.yaml]: I knew I had to reread it...

**Comment by 0xlildoudou** [comment]
Thanks :)

**Comment by ocervell** [comment]
Apart from the typo namager --> manager, it looks fine to me.

**Comment by 0xlildoudou** [comment]
Hi @ocervell ,

I have change my error

**Comment by ocervell** [comment]
I've tested it on Minikube and it's working great ! Thank you so much for contributing to `secator`.
 
I'm going to merge this soon.

Here are potential improvements that could be made in the future:
- Make an optional dependency on Redis Bitnami chart (thus removing the redis-service.yaml and redis.yaml)
- Make an optional dependency on RabbitMQ Bitnami chart (to supporter another broker)
- Make an optional dependency on MongoDB Bitnami chart (to support the `mongodb` addon)
- Make a ConfigMap to customize secator's configuration `config.yml` file
- Add an HPA for worker autoscaling (create new workers depending on the size of the Redis queue)
- Add resources / requests limits to secator deployments

Cheers !!

**Comment by 0xlildoudou** [comment]
Hi,

Thanks 😀

don't worry, I'll be able to make changes when I have time 😁

**Pull request merged**